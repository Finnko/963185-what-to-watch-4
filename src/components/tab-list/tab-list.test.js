import React from 'react';
import renderer from 'react-test-renderer';
import TabList from './tab-list.tsx';

const props = {
  activeTab: `Overview`,
  onTabClick: jest.fn(),
  tabNames: [`Something`, `Something Else`],
};

describe(`TabList component render correctly`, () => {
  it(`Should TabList component render correctly`, () => {
    const tree = renderer
      .create(
          <TabList
            {...props}
          />
      ).toJSON();

    expect(tree).toMatchSnapshot();
  });
});
