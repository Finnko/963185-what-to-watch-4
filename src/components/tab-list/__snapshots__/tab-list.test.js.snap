// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`TabList component render correctly Should TabList component render correctly 1`] = `
<ul
  className="movie-nav__list"
>
  <li
    className="movie-nav__item "
  >
    <a
      className="movie-nav__link"
      href="#"
      onClick={[Function]}
    >
      Something
    </a>
  </li>
  <li
    className="movie-nav__item "
  >
    <a
      className="movie-nav__link"
      href="#"
      onClick={[Function]}
    >
      Something Else
    </a>
  </li>
</ul>
`;
